serverless/sql-databases/reference-content/known-differences.mdx

```diff
@@ -84,7 +84,7 @@ If you require strict PostgreSQL features compatibility, you can use [Managed da
     RESET timezone;
     ```
 
-- Security labels cannot be defined or changed. Example:
+- Security labels cannot be defined or changed.
 
     ```
     SECURITY LABEL FOR provider ON TABLE table IS 'label';
```

Comment:
Suggestion for part title: "Unsupported Configuration commands" (easier to read / eye catching when going through the page quickly)